src/CabanaPD_Solver.hpp

```diff
@@ -148,8 +148,16 @@ class SolverElastic
         neighbors = std::make_shared<neighbor_type>( x, 0, particles->n_local,
                                                      force_model.delta, 1.0,
                                                      mesh_min, mesh_max );
-        int max_neighbors =
+        unsigned max_neighbors;
```

Comment:
@streeve : Is this line kind of initialization? (same for Line 154)

Reply:
Yes